testing/postgres-client-tests/c/postgres-c-connector-test.c

```diff
@@ -0,0 +1,137 @@
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <libpq-fe.h>
+
+#define QUERIES_SIZE 13
+
+char *queries[QUERIES_SIZE] = {
+    "create table test (pk int, value int, d1 decimal(9, 3), f1 float, primary key(pk))",
```

Comment:
Should have a string type in these tests as well

Reply:
Got it, I'll add string types in all the client tests.